lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -65,6 +65,15 @@ function runA11yChecks() {
         // avoid circular JSON concerns
         node.element = node.any = node.all = node.none = undefined;
       });
+
+      // Ensure errors can be serialized over the protocol
```

Comment:
to connect the dots here, in the #10644 screenshot the reason `incompleteResult.error` only had an `errorNode` property and no others is because `error` is actually an `Error` and I guess axe adds `errorNode` to the instance?

Reply:
correct and `Error`s don't serialize their properties though I have always wondered why no one wants to change that :)